tests/test_jobresource.py

```diff
@@ -16,6 +18,7 @@ def _test_prints_objects(self, cmd_mod, resource_name):
         jobid = '1/2/3'
         with mock.patch.object(cmd_mod, 'get_job', autospec=True) as mock_gj:
             # Patch job.items.iter_json() to return our objects
+            mock_gj.return_value._metadata_updated = time.time()
```

Comment:
There was a problem with mocks that got uncovered when being ported to py3, can't remember now, let me see if i can find it quickly.